clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/model/EcsServerGroupEventStatus.java

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.clouddriver.ecs.model;
+
+public enum EcsServerGroupEventStatus {
+
+  Success,
+  Failure,
+  Transition
```

Comment:
Would like to see these `CAPITALIZED` to follow convention of the rest of the projects.

Reply:
A number of enums don't use full capitalization such as KubernetesPatchOptions.MergeStrategy, com.netflix.spinnaker.clouddriver.security.ProviderVersion, KubernetesDeployManifestDescription.Source, com.netflix.titus.grpc.protogen.ScalingPolicyState, etc.  I'd like to avoid modifying the capitalization left and right and have it in the right capitalization from the start.  What do you think?

Reply:
Booooo, I wasn't aware we weren't staying consistent on this. Fine as is, then. Thanks for pointing this out!